src/app/mood/page.js

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import emo1 from "../../public/images/emo1.png";
+import emo2 from "../../public/images/emo2.png";
+import emo3 from "../../public/images/emo3.png";
+import emo4 from "../../public/images/emo4.png";
+import styles from "./page.module.css";
+import { josefinSans } from "@/utils/fonts";
+import Exit from "@/components/Exit";
+import emo5 from "@/public/images/emo5.png";
```

Comment:
next edit we should move this so they're all in line? or maybe we make this an array if possible?